masterlist.yaml

```
@@ -6644,27 +6644,72 @@ plugins:
+        condition: 'active("xulCliffsOfAnvil.esp") and not active("ULE KragenirsDeathQuest-CliffsOfAnvil patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Unique Landscapes - Cloudtop Mountains'
+          - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
+        condition: 'active("xulCloudtopMountains.esp") and not file("ULE KragenirsDeathQuest-CloudtopMountains patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Unique Landscapes - Rolling Hills'
+          - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
+        condition: 'active("xulRollingHills_EV.esp") and not active("ULE KragenirsDeathQuest-RollingHills patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Unique Landscapes - The Eastern Peaks'
+          - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
+        condition: 'active("xulTheEasternPeaks.esp") and not file("ULE KragenirsDeathQuest-EasternPeaks patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Unique Landscapes'
+          - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
+        condition: 'active("xulBeachesOfCyrodiilLostCoast.esp") and active("xulCliffsOfAnvil.esp") and active("xulCloudtopMountains.esp") and active("xulRollingHills_EV.esp") and active("xulTheEasternPeaks.esp") and not active("ULE KragenirsDeathQuest-UniqueLandscapes merged patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Unique Landscapes'
+          - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
+        condition: 'active("Unique Landscapes.esp") and not active("ULE KragenirsDeathQuest-UniqueLandscapes merged patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Let the People Drink' ]
+        condition: 'active("(LetThePeopleDrink.esp|Better Cities Resources.esm)") and not active("KragenirsDeathQuest-LetThePeopleDrink patch.esp")'
     tag: [ Invent.Add ]
     dirty:
       - <<: *quickClean
-        crc: 0x028F29D6
-        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        crc: 0xF427CF43
+        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 1
   - name: 'KDQ - Rural Line Additions.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/26219/' ]
+    msg: [ *optional ]
     clean:
       - crc: 0x86EC5DE6
-        util: 'TES4Edit v4.0.4'
-  - name: 'KragenirsDeathQuest-LostCoast patch.esp'
-    clean:
-      - crc: 0xD92DDDE2
-        util: 'TES4Edit v4.0.4'
+        util: 'TES4Edit v4.0.4b'
   - name: 'KragenirsDeathQuest-LetThePeopleDrink patch.esp'
+    req:
+      - name: 'LetThePeopleDrink.esp'
+        condition: 'not active("Better Cities Resources.esm")'
```

Adjusting this condition to reflect the BC compatibility comments I made previously.

```suggestion
        condition: 'not active("Better (Cities IC Arboretum|Imperial City)\.esp")'
```